image/image.go

```diff
@@ -112,21 +112,21 @@ func (img *Image) MarshalJSON() ([]byte, error) {
 
 // History stores build commands that were used to create an image
 type History struct {
-	// Created timestamp for build point
+	// Created is the timestamp at which the image was created
 	Created time.Time `json:"created"`
-	// Author of the build point
+	// Author is the name of the author that was specified when building the image
```

Comment:
Should probably be `committing` as well here, given that this value is set by the `--author` flag on `docker commit` https://docs.docker.com/engine/reference/commandline/commit/